Jenkinsfile-rf-pipeline

```diff
@@ -0,0 +1,308 @@
+                                string(defaultValue: 'bpeck/jenkins-continuous-infra.apps.ci.centos.org@FEDORAPROJECT.ORG', description: 'Principal for authenticating with fedora build system', name: 'FEDORA_PRINCIPAL'),
+                                booleanParam(defaultValue: false, description: 'Force generation of the image', name: 'GENERATE_IMAGE'),
+                        ]
+                ),
+        ]
+)
+
+podTemplate(name: 'fedora-atomic-inline', label: 'fedora-atomic-inline', cloud: 'openshift', serviceAccount: 'jenkins',
+        idleMinutes: 1,  namespace: 'continuous-infra',
+        containers: [
+                // This adds the custom slave container to the pod. Must be first with name 'jnlp'
+                containerTemplate(name: 'jnlp',
+                        image: '172.30.254.79:5000/continuous-infra/jenkins-continuous-infra-slave',
+                        ttyEnabled: false,
+                        args: '${computer.jnlpmac} ${computer.name}',
+                        command: '',
+                        workingDir: '/tmp'),
+        ])
+
+node('fedora-atomic-inline') {
+    ansiColor('xterm') {
+        timestamps {
+            def currentStage = ""
+            try {
+                deleteDir()
+
+                // Set our default env variables
+                pipelineUtils.setDefaultEnvVars()
+
+                // Parse the CI_MESSAGE and inject it as env vars
+                pipelineUtils.injectFedmsgVars()
+
+                // Set our current stage value
+                currentStage = "ci-pipeline-rpmbuild"
+                stage(currentStage) {
+
+                    // SCM
+                    dir('ci-pipeline') {
+                        git 'https://github.com/CentOS-PaaS-SIG/ci-pipeline'
+                    }
+                    dir('cciskel') {
+                        git 'https://github.com/cgwalters/centos-ci-skeleton'
+                    }
+                    dir('sig-atomic-buildscripts') {
+                        git 'https://github.com/CentOS/sig-atomic-buildscripts'
+                    }
+
+                    // Set stage specific vars
+                    pipelineUtils.setStageEnvVars(currentStage)
+
+                    //Set our message topic, properties, and content
+                    (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields("package.running")
+
+                    // Send message org.centos.prod.ci.pipeline.package.running on fedmsg
+                    sendMessage(messageProperties, messageContent)
+
+                    // Provision of resources
+                    pipelineUtils.provisionResources(currentStage)
+
+                    // Stage resources - RPM build system
+                    pipelineUtils.setupStage(current_stage, 'fedora-atomic-key')
+
+                    // Rsync Data
+                    pipelineUtils.rsyncData(currentStage)
```

Comment:
You changed the name of the method so this needs to be updated

Reply:
rsyncData() didn't change name. The method rsyncResults was renamed to runTaskAndReturnLogs(). 
This wouldn't need to be updated.

Reply:
Then why do we need runTaskAndReturnLogs() for?

Reply:
In it's current form rsyncData() creates the text content for the task.env file based on the current stage in the jenkins file and then calls the runTaskAndReturnLogs() method which handles the actual rsyncing of the task.env to the duffy host and the subsequent rsyncing of the logs to the jenkins workspace.

The two methods could be combined into a single one potentially, but it would require changes in the groovy files in var/  (see 54-70 in vars/ostreeBootSanity for example)

Let me know if you'd like to have those changes made.

Reply:
No I get what you did sounds fine to me